build/helper/metadata_add_all.py

```diff
@@ -213,6 +215,24 @@ def add_all_attribute_metadata(attributes, config):
     return attributes
 
 
+def add_all_metadata(functions, attributes, enums, config):
```

Comment:
add docstring, should say that this modifies config since it's a little strange that it uses it and also modifies it.